<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>810</width>
    <height>592</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="login">
       <widget class="QWidget" name="formLayoutWidget">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>220</y>
          <width>211</width>
          <height>141</height>
         </rect>
        </property>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineLogin"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="Password">
           <property name="text">
            <string>Password</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="linePassword"/>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="Login">
           <property name="text">
            <string>Login</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0" colspan="2">
          <widget class="QPushButton" name="pushEntrar">
           <property name="text">
            <string>Entrar</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0" colspan="2">
          <widget class="QPushButton" name="pushCadastrar">
           <property name="text">
            <string>Cadastrar</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="cadastro">
       <widget class="QWidget" name="formLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>160</y>
          <width>341</width>
          <height>201</height>
         </rect>
        </property>
        <layout class="QFormLayout" name="formLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="labelNome">
           <property name="text">
            <string>Nome</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="labelLoginCadastro">
           <property name="text">
            <string>Login</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="labelSenha">
           <property name="text">
            <string>Senha</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="labelConfirmarSenha">
           <property name="text">
            <string>Confirmar senha</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineNome"/>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="lineLoginCadastro"/>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="lineSenha"/>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="lineConfirmarSenha"/>
         </item>
         <item row="4" column="0" colspan="2">
          <widget class="QPushButton" name="pushEnviar">
           <property name="text">
            <string>Enviar</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0" colspan="2">
          <widget class="QPushButton" name="pushVoltar">
           <property name="text">
            <string>Voltar</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="principal"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>810</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
